glsk/glsk-document-cim/src/main/java/com/powsybl/glsk/cim/CimGlskPeriod.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.glsk.cim;
+
+import com.powsybl.glsk.api.AbstractGlskPoint;
+import org.threeten.extra.Interval;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * CIM type glsk internal object: a period contains a list of GlskPoint
+ * @author Pengbo Wang {@literal <pengbo.wang@rte-international.com>}
+ */
+public class CimGlskPeriod {
+
+    /**
+     * Interval of Period
+     */
+    private Interval periodInterval;
+    /**
+     * Resolution of point : ex. PT60M = 60 minutes
+     */
+    private String resolution;
+    /**
+     * CurveType: A01 and A03
+     */
+    private String curveType;
+    /**
+     * List of GlskPoint in the Period
+     */
+    private List<AbstractGlskPoint> glskPoints;
+
+    /**
+     * @param element Dom element
+     * @param subjectDomainmRID Country mRID
+     * @param curveType curveType
+     */
+    public CimGlskPeriod(Element element, String subjectDomainmRID, String curveType) {
+        Objects.requireNonNull(element);
+        String intervalStart = element.getElementsByTagName("start").item(0).getTextContent();
+        String intervalEnd = element.getElementsByTagName("end").item(0).getTextContent();
+        this.periodInterval = Interval.parse(intervalStart + "/" + intervalEnd);
```

Comment:
Instant.parse throw a DataTimeParseException at timestamp with format like "2018-08-28T22:00Z" because of the timezone . I think that is why they parsed it with Interval.